apps/opik-backend/src/test/java/com/comet/opik/infrastructure/redis/RedisUrlParserTest.java

```diff
@@ -0,0 +1,37 @@
+package com.comet.opik.infrastructure.redis;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Do we have some wildcard import policy in opik? In EM we were trying to avoid them